.travis.yml

```diff
@@ -0,0 +1,6 @@
+language: node_js
+node_js: "12"
```

Comment:
Why do you pass `node_js` as a string?

Reply:
I followed [documentation](https://docs.travis-ci.com/user/languages/javascript-with-nodejs/#specifying-nodejs-versions) and it's mostly because you can specify it not just like a number. You can use these values too:

```
node latest stable Node.js release
lts/* latest LTS Node.js release
```

But numbers work too. It doesn't matter.